be/test/olap/rowset/segment_v2/column_reader_writer_test.cpp

```diff
@@ -58,6 +58,7 @@ void test_nullable_data(uint8_t* src_data, uint8_t* src_is_null, int num_rows, s
 
         ColumnWriterOptions writer_opts;
         writer_opts.encoding_type = encoding;
+        writer_opts.compression_type = segment_v2::CompressionTypePB::LZ4F;
```

Comment:
make LZ4F the default value for ColumnWriterOptions.compression_type so that we don't need to set it everywhere

Reply:
OK